### ISSUE TITLE
l10n.t() overrides not picked up by export

### ISSUE THREAD

**Issue by asr1** [microsoft/vscode#176561]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 
- OS Version: 

Steps to Reproduce:

1. Follow the readme [here](https://github.com/microsoft/vscode-extension-samples/tree/main/l10n-sample)
2. In Extension.ts, add a call to `l10n.t("test2");` using only the simple string override.
3. In Extension.ts, add a call to `l10n.t()` using the override that takes message, args, and comment:
```javascript
l10n.t({                    
message: `Hello: ${name}`,
 args: ["Bob"],
comment: ["{name} is the name of the user."],                });`
```
4. Run npx @vscode/l10n-dev export -o ./l10n ./src
5. Expect both "test2" and "Hello ${name}" to be produced in the output file.
6. Actual: Only "test2" appears in the generated bundle.json file.

**Comment by TylerLeonhardt** [comment]
Interpolated strings don't work because we are extracting outside of a running build. We don't know what `name` will end up being.

The correct syntax is:
```ts
l10n.t({                    
    message: "Hello: {0}",
    args: ["Bob"],
    comment: ["{0} is the name of the user."]
});
```

**Comment by asr1** [comment]
Wonderful, that fixed the issue, thank you!

**Comment by asr1** [comment]
@TylerLeonhardt I have one other question: I have created a `bundle.l10n.ja.json` file, but when I change my browser's language to Japanese, I'm not seeing the translations from this file appear. It seems to be taking the untranslated string that I pass in (not even the output of the `bundle.l10n.json` file. 

I've configured my l10 property in my package.json appropriately (the `npx @vscode/l10n-dev export -o ./l10n ./src` command does not complain about my outdir not matching). Is there a step I could be missing here?.
